packages/build/src/local/repository-status.ts

```diff
@@ -52,7 +52,7 @@ export function getRepositoryStatus(
 
   const result: RepositoryStatus = {
     clean: true,
-    hasUnpushedTags: (tagStatus.stdout?.trim() ?? '' + tagStatus.stderr?.trim() ?? '') !== 'Everything up-to-date'
+    hasUnpushedTags: ((tagStatus.stdout?.trim() ?? '') + (tagStatus.stderr?.trim() ?? '')) !== 'Everything up-to-date'
```

Comment:
That way it will be hard to mess up again by accident

```suggestion
    hasUnpushedTags: `${tagStatus.stdout?.trim() ?? ''}${tagStatus.stderr?.trim() ?? ''}` !== 'Everything up-to-date'
```